openapi/data_repository_service.swagger.yaml

```diff
@@ -301,34 +188,63 @@ definitions:
       mime_type:
         type: string
         description: |-
-          A string providing the mime-type of the Data Object.
+          A string providing the mime-type of the `Object`.
         example:
           application/json
       checksums:
         type: array
```

Comment:
Add `minItems : 1` to ensure at least one checksum is provided.

Reply:
Done -- pls resolve if it LGTY